sass/header/header.sass

```
@@ -0,0 +1,266 @@
+@import ../mixins.sass
+
+@import ./vars_header.sass
+
+@mixin header
+    .header
+        &__container
+            background-image: url('../../assets/bg1.png')
+            max-width: 100%
+            height: 223px
+            background-repeat: no-repeat
+            background-size: cover
+            .header__head
```

```suggestion
    .header
        &__container
            background-image: url('../../assets/bg1.png')
            max-width: 100%
            height: 223px
            background-repeat: no-repeat
            background-size: cover
        &__head
```
Tránh nesting quá nhiều
Thay vì selector .header__container .header__head
Thì chỉ cần .header__head thôi